### ISSUE TITLE
a11y: Semantics for reaction chips in the message list

### ISSUE THREAD

**Issue by chrisbobbe** [zulip/zulip-flutter#434]
We should give appropriate direction to screen-reader software (such as Apple's VoiceOver) in the area below a message that shows message reactions, in the message list.

Probably the user should be able to select that whole area and hear "Message reactions"—or perhaps that plus e.g. "(10 total)"—then be able to focus each chip in turn, hearing e.g. "[emoji name]: you and 3 others: [list of names]". Then there's probably some structured way to tell the software that it's a button, along with (hopefully) what will happen if it's pressed; and the screen reader will turn that into the right speech for the user.

**Comment by gnprice** [comment]
I believe this was resolved by b2321839f5317cd70878e1ee5359418f6e0e2567 and 41e3d57f2cde5b438fbfb73d4867b3077e2bef3c, in #1700.

@chrisbobbe does that seem right to you? If not, we can reopen.

**Comment by chrisbobbe** [comment]
Yep, that seems right! We can file a new issue if it turns out to be buggy in some way.
